firefox-ios/Client/Frontend/Share/ShareManager.swift

```diff
@@ -9,39 +9,28 @@ import WebKit
 import UniformTypeIdentifiers
```

Comment:
This file was the main refactor. Lots of old cruft cut, improved interface, and better separation of concerns.